pkg/agent/controller/networkpolicy/l7engine/reconciler.go

```diff
@@ -383,6 +389,23 @@ func (r *Reconciler) startSuricata() {
 	// /etc/suricata/suricata.yaml.
 	suricataAntreaConfigData := fmt.Sprintf(`%%YAML 1.1
 ---
+outputs:
+  - fast:
+      enabled: yes
+      filename: fast.log
+      append: yes
+  - eve-log:
+      enabled: yes
+      filetype: regular
+      filename: eve.json
+      pcap-file: false
+      community-id: false
+      community-id-seed: 0
+      xff:
+        enabled: no
+      types:
+        - alert:
+            tagged-packets: yes
```

Comment:
Will they rotate automatically? BTW, does np.log rotate?

Reply:
Haven't set the rotation, it can be set if needed. np.log rotates based on Lumberjack.

Reply:
Added [event rotation](https://suricata.readthedocs.io/en/suricata-6.0.0/output/eve/eve-json-output.html#output-eve-rotate).